be-devops-engineer-challenge-heating-control.md

```diff
@@ -0,0 +1,54 @@
+
+# WATTx Software Engineer Challenge: **Heating Control**
+
```

Comment:
Please Note: The challenge contains two parts: 
- 1. Implementation
- 2. DevOps